No longer publish or overwrite any "latest" bundles on CDN (#5697)
plotly/plotly.js: README.md

### Use the plotly.js CDN hosted by Fastly

#### A minified plotly.js X.Y.Z release
```html
<script src="https://cdn.plot.ly/plotly-1.58.4.min.js" charset="utf-8"></script>
```

#### An un-minified version is also available
```html
<script src="https://cdn.plot.ly/plotly-1.58.4.js" charset="utf-8"></script>
```

and use the `Plotly` object in the window scope.

> ##### Please note that after v2 the "plotly-latest" outputs (e.g. https://cdn.plot.ly/plotly-latest.min.js) will no longer be updated on the CDN and stay at last v1 patch i.e. v1.58.4. Therefore, in order to upgrade to plotly.js v2 and higher, you may consider switching to a specific plotly.js version.

Reviewer: ```suggestion
##### Please note that as of v2 the "plotly-latest" outputs (e.g. https://cdn.plot.ly/plotly-latest.min.js) will no longer be updated on the CDN, and will stay at the last v1 patch v1.58.4. Therefore, to use the CDN with plotly.js v2 and higher, you must specify an exact plotly.js version.
```

Fastly supports Plotly.js with free CDN service. Read more at <https://www.fastly.com/open-source>

> ### Download the latest release or a release candidates (rc)

Reviewer: ```suggestion
### Download the latest release or a release candidate (rc)
```
